configure.ac

```diff
@@ -27,7 +27,7 @@ dnl     checks for compiler characteristics
 dnl     checks for library functions
 dnl     checks for system services
 
-m4_include([version.m4])
+m4_include([m4/version.m4])
```

Comment:
Remember to update the release checklist wiki page for this new location.

Reply:
Will do